src/sider/sykepengsoknader/soknad-felles-oppsummering/Oppsummeringsvisning.tsx

```diff
@@ -34,9 +35,10 @@ const Oppsummeringsvisning = ({
       {sporsmal
         .filter(
           (sporsmal) =>
-            sporsmal.svar.length > 0 ||
-            sporsmal.undersporsmal.length > 0 ||
-            sporsmal.svartype === SvarTypeDTO.IKKE_RELEVANT
+            (sporsmal.svar.length > 0 ||
+              sporsmal.undersporsmal.length > 0 ||
+              sporsmal.svartype === SvarTypeDTO.IKKE_RELEVANT) &&
+            !erVaerKlarOverAt(sporsmal)
```

Comment:
Alle bruker denne komponenten for å vise oppsummering, så tenkte det var fint å filtrere ut her.